starboard/android/shared/player_components_factory.cc

```diff
@@ -44,7 +44,11 @@ bool PlayerComponents::Factory::OutputModeSupported(
       return true;
     }
     DrmSystem* android_drm_system = static_cast<DrmSystem*>(drm_system);
-    return !android_drm_system->require_secured_decoder();
+    bool is_widevine_l1 = android_drm_system->require_secured_decoder();
+    SB_LOG_IF(ERROR, is_widevine_l1)
+        << "Invalid output mode due to DRM Widevine L1 videos "
```

Comment:
Instead of using "Invalid output mode", consider using "*** isn't supported because secure decoder is required" (please rephrase).

Reply:
Done.